sql/activesets/activesets_test.go

```diff
@@ -49,3 +49,29 @@ func TestActiveSet(t *testing.T) {
 	require.ErrorIs(t, err, sql.ErrNotFound)
 	require.Empty(t, blob)
 }
+
+func TestCachedActiveSet(t *testing.T) {
```

Comment:
Same here, the test doesn't check if `sql.WithQueryCache` actually causes the results of a query to be cached.

Reply:
Fixed by checking the query counter